specs/proposals.md

```diff
@@ -24,15 +28,39 @@ are part of later specification milestones.
 
 [cannon]: https://github.com/ethereum-optimism/cannon
 
-## Proposing L2 output commitments
+## Definitions
 
-The proposer's role is to construct and submit output commitments on a configurable interval to a contract on L1.
+### Constants
 
-TODO: describe integration with rollup node and L2 execution engine (see PR #179).
+| Name                   | Value | Unit    |
+| ---------------------- | ----- | ------- |
+| `SUBMISSION_FREQUENCY` | `100` | seconds |
```

Comment:
I think submission frequency should be more like `1800` seconds. The difference between withdrawals happening in 7 days vs in 7 days and 30 mins makes the $ cost of submission more of a limiting factor here.

If we had a shorter withdrawal period then reducing this number might matter, but until then I think 30 mins or more is super reasonable.